pdm/test/action.yml

```diff
@@ -55,6 +57,13 @@ runs:
         echo "suffix=.${id}" >> $GITHUB_OUTPUT
       shell: bash
 
+    - name: Start and wait Docker Compose
+      if: inputs.docker-compose == 'true' and report-only == null
```

Comment:
GitHub is complaining:
```text
Download action repository 'LedgerHQ/actions@main' (SHA:88edd78e360e22a68c7a2b04ba9e36829b45a7ca)
Error: LedgerHQ/actions/main/pdm/test/action.yml (Line: 61, Col: 11): Unexpected symbol: 'and'. Located at position 33 within expression: inputs.docker-compose == 'true' and report-only == null
Error: GitHub.DistributedTask.ObjectTemplating.TemplateValidationException: The template is not valid. LedgerHQ/actions/main/pdm/test/action.yml (Line: 61, Col: 11): Unexpected symbol: 'and'. Located at position 33 within expression: inputs.docker-compose == 'true' and report-only == null
   at GitHub.DistributedTask.ObjectTemplating.TemplateValidationErrors.Check()
   at GitHub.Runner.Worker.ActionManifestManager.ConvertRuns(IExecutionContext executionContext, TemplateContext templateContext, TemplateToken inputsToken, String fileRelativePath, MappingToken outputs)
   at GitHub.Runner.Worker.ActionManifestManager.Load(IExecutionContext executionContext, String manifestFile)
Error: Failed to load LedgerHQ/actions/main/pdm/test/action.yml
```

I guess this should be `... && inputs.report-only == null` (using `&&` instead of `and`, and adding `inputs.`.